opbeat/base.py

```diff
@@ -416,6 +415,25 @@ def _get_transport(self, parsed_url):
             self._transports[parsed_url] = self._transport_class(parsed_url)
         return self._transports[parsed_url]
 
+    def _filter_exception_type(self, data):
+        exception = data.get('exception')
+        if not exception:
+            return False
+
+        exc_type = exception.get('type')
+        exc_module = exception.get('module')
+        if exc_module == 'None':
+            exc_module = None
+
+        for exc_to_filter in self.filter_exception_types:
+            exc_to_filter_type = exc_to_filter.split(".")[-1]
```

Comment:
could it make sense to split the `filter_exception_types` list into modules/classes in `__init__` instead of doing it on every error?